### PR TITLE
End point to get all todo items

### PR DESCRIPTION
**Pull request by vinay-jtc**
### Fatch to all todo items.

URI: GET /todos
Response HTTP Status: 200
Response body: [{ id: …, title: … }, ...]
## Database schema changes

- NA

### Automated test cases added

- Should fetch all the todo items with response 200 
- Return 200 if array is empty

### Manual test cases run

- Done

### CODE REVIEW

File: src/models/index.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { TodoItem } from './todo-item';
+import { TodoItems } from './todo-items';
 import { BaseModel } from './base-model';
 import { ModelFactory } from './model-factory';
```
Comment (coder1033) [inline src/models/index.ts:2]: Order alphabetically.

File: src/models/index.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
 export {
   TodoItem,
   BaseModel,
   ModelFactory,
+  TodoItems
```
Comment (coder1033) [inline src/models/index.ts:10]: Order Alphabetically.

**Review by coder1033 — APPROVED** [review]

**Comment by coder1033** [comment]
Merge after addressing the comments.

File: src/models/index.ts (hunk 2 above)
Comment (vinay-jtc) [reply src/models/index.ts:10]: Updated

**Review by vinay-jtc — COMMENTED** [review]
Updated comment and now merging.

**Pull request merged**